pkg/upgrade/upgrade.go

```diff
@@ -179,22 +185,9 @@ func CreateDefaultDSC(cli client.Client, platform deploy.Platform) error {
 	case err == nil:
 		fmt.Printf("created DataScienceCluster resource")
 	case apierrs.IsAlreadyExists(err):
-		// Update if already exists
-		if platform == deploy.ManagedRhods {
-			fmt.Printf("DataScienceCluster resource already exists in Managed. Updating it.")
-			data, err := json.Marshal(releaseDataScienceCluster)
-			if err != nil {
-				return fmt.Errorf("failed to get DataScienceCluster custom resource data: %v", err)
-			}
-			err = cli.Patch(context.TODO(), releaseDataScienceCluster, client.RawPatch(types.ApplyPatchType, data),
-				client.ForceOwnership, client.FieldOwner("opendatahub-operator"))
-			if err != nil {
-				return fmt.Errorf("failed to update DataScienceCluster custom resource:%v", err)
-			}
-		} else {
-			fmt.Printf("DataScienceCluster resource already exists. It will not be updated with default DSC.")
-			return nil
-		}
+		// Do not update the DSC if it already exists
```

Comment:
@etirelli Can you confirm for Managed service, we would not be patching with default DSC if the instance is already created?

Reply:
i am fine with the PR as long as we can confirm ^

Reply:
@VaishnaviHire correct, that is my understanding. If a DSC already exists, we should preserve it.